schematools/contrib/django/management/commands/change_dataset.py

```diff
@@ -0,0 +1,110 @@
+from argparse import ArgumentTypeError
+
+from django.core.exceptions import ValidationError
+from django.core.management import BaseCommand, CommandError
+
+from schematools.contrib.django.models import Dataset
+
+
+def str2bool(v):
```

Comment:
Dit zit toch in distutils? (distutils.util.strtobool)

Reply:
oh, cool, weer wat geleerd!